DevREADME.md

```diff
@@ -99,4 +99,132 @@ You may encounter the failure of npm install, the error message would say:
 Error installing vscode.d.ts: Error: read ECONNRESET
 If you encounter this problem, maybe your NODE is too old, in version v10.13.0 it 
 can work well.
-or run "npm cache clean -force"
\ No newline at end of file
+or run "npm cache clean -force"
+
+---
+
+## Style Guide
+Below are some general code guidelines that should be followed when adding or updating the code base. It is the expectation that participating members actively apply this style guide to legacy code during maintenance so we can eventually have a noticeably consistent developer experience.
+### Names
+1. Use `PascalCase` for type/interface names, enums, setters, getters and global singletons
+2. Use `camelCase` for function, argument and local variable names.
+3. Use `UPPER_CASE` for global constants.
+4. Use `_` as a prefix for private variables
+5. Use `I` as a prefix for interface names that define function declarations.
+5. Use whole words in names where possible.
+
+### Types
+1. Do not export types/functions unless you need to share them across multiple components.
+2. Within a file, type definitions should come first.
+
+### Comments
+1. Use JSDoc-style comments for functions, interfaces, enums, and classes.
+2. Limited to using an informal description and @type/@param/@return when applicable
+
+### General
```

Comment:
- Add item to prefer "let" over "var" 
- Add item related to const. However, I don't really know exactly what to say about const. I've personally started using it to declare just about everything to purposefully cause errors and make me rethink what/why/how I am doing something.

Reply:
which item is for `let` and for `const`?

Reply:
The first bullet is standalone, but we could just say both items would read
_Use const or let instead of var_

I was just remarking that a clearer definition of when to use const might be helpful because I do find that it has been helping me work evolve my plans